lib/configs/browser.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
     'github/no-blur': 'error',
     'github/no-dataset': 'error',
     'github/no-innerText': 'error',
-    'github/unescaped-html-literal': 'error'
+    'github/unescaped-html-literal': 'error',
+    'github/no-useless-passive': 'error',
+    'github/no-non-passive-high-frequency-events': 'error',
+    'github/prefer-observers-over-events': 'error'
```

Comment:
We can tighten this up to something like `prefer-observers`. Lint rules typically name the alternatives in the error message rather than the rule name.